new_bug.cgi

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/perl -T
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+#
+# Contributor(s): Sebastin Santy <sebastinssanty@gmail.com>
+#
+##############################################################################
+#
+# new_bug.cgi
+# -------------
+# Single page interface to file bugs
+#
+##############################################################################
+
+use 5.10.1;
+use strict;
+use warnings;
+
+use lib qw(. lib local/lib/perl5);
+
+use Bugzilla;
+use Bugzilla::Constants;
+use Bugzilla::Util;
+use Bugzilla::Error;
+use Bugzilla::Bug;
+use Bugzilla::User;
+use Bugzilla::Hook;
+use Bugzilla::Product;
+use Bugzilla::Classification;
+use Bugzilla::Keyword;
+use Bugzilla::Token;
+use Bugzilla::Field;
+use Bugzilla::Status;
+use Bugzilla::UserAgent;
+
+my $user = Bugzilla->login(LOGIN_REQUIRED);
+
+my $cloned_bug;
+my $cloned_bug_id;
+
+my $cgi = Bugzilla->cgi;
+my $dbh = Bugzilla->dbh;
+my $template = Bugzilla->template;
+my $vars = {};
+
+my $format = $template->get_format("bug/new_bug",
+                                   scalar $cgi->param('format'), 
+                                   scalar $cgi->param('ctype'));
+
```

Comment:
I think we can avoid the whole "get_format" thing.

$template->process("bug/new_bug.html.tmpl", ...);